site/content/api/Tenant-API.md

```diff
@@ -0,0 +1,264 @@
+| *reply-to*       | yes       | *properties*             | *string*     | MUST contain the source address that the client wants to received response messages from. This address MUST be the same as the source address used for establishing the client's receive link (see [Preconditions]({{< relref "#preconditions" >}})). |
+| *tenant-id*      | yes       | *application-properties* | *string*     | MUST contain the ID of the tenant that is subject to the operation. |
+
+## Standard Response Properties
+
+The following table provides an overview of the properties shared by all response messages regardless of the particular operation being invoked.
+
+| Name             | Mandatory | Location                 | Type         | Description |
+| :--------------- | :-------: | :----------------------- | :----------- | :---------- |
+| *correlation-id* | yes       | *properties*             | *message-id* | Contains the *message-id* (or the *correlation-id*, if specified) of the request message that this message is the response to. |
+| *tenant-id*      | yes       | *application-properties* | *string*     | Contains the ID of the tenant that is subject of the outcome of the operation. |
+| *status*         | yes       | *application-properties* | *int*        | Contains the status code indicating the outcome of the operation. Concrete values and their semantics are defined for each particular operation. |
+
+# Delivery States
+
+Hono uses the following AMQP message delivery states when receiving request messages from clients:
+
+| Delivery State | Description |
+| :------------- | :---------- |
+| *ACCEPTED*     | Indicates that Hono has successfully received and accepted the request for processing. |
+| *REJECTED*     | Indicates that Hono has received the request but was not able to process it. The *error* field contains information regarding the reason why. Clients should not try to re-send the request using the same message properties in this case. |
+
+# Tenant Format
+
+Most of the operations of the Tenant API allow or require the inclusion of tenant data in the payload of the
+request or response messages of the operation. Such payload is carried in the body of the corresponding AMQP 
+messages as part of a single *AMQP Value* section.
+
+The tenant data is carried in the payload as a UTF-8 encoded string representation of a single JSON object. It is an error to include payload that is not of this type.
+
+The table below provides an overview of the standard members defined for the JSON object:
+
+| Name                     | Mandatory | Type       | Default Value | Description |
+| :------------------------| :-------: | :--------- | :------------ | :---------- |
+| *tenant-id*              | *yes*     | *string*   |               | The ID of the tenant. |
+| *enabled*                | *no*      | *boolean*  | *true*        | If set to *false* the tenant is currently disabled.|
```

Comment:
literal values should be put between single back quotes, e.g. `true`
this is currently wrong in the Credentials API docs as well.

Reply:
fixed